### PR TITLE
[DOCU-3153] Contextual Developer Analytics

### PR DESCRIPTION
**Pull request by Guaris**
Contextual Developer Analytics PR

https://konghq.atlassian.net/jira/software/c/projects/DOCU/boards/58?modal=detail&selectedIssue=DOCU-3153

https://deploy-preview-5683--kongdocs.netlify.app/konnect/dev-portal/
https://deploy-preview-5683--kongdocs.netlify.app/konnect/dev-portal/customization/
https://deploy-preview-5683--kongdocs.netlify.app/konnect/dev-portal/applications/application-overview/

### CODE REVIEW

File: app/konnect/dev-portal/index.md [hunk 1]
```diff
@@ -40,8 +40,24 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+The Developer Portal supports built-in analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This feature gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable **Dashboard**. Each application has it's own dashboard that provides a high-level summary of **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points:
```
Comment (lena-larionova) [inline app/konnect/dev-portal/index.md:45]:
```suggestion
You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard. 

Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
```
Rephrased some repetitive wording.

 "Each application has its own dashboard that provides" doesn't work - Either:
* "Each application has a dashboard that provides"
* "Each application has its own dashboard, which provides"

I suggested the "which" phrasing, but you're welcome to go with the other option.

File: app/konnect/dev-portal/index.md [hunk 2]
```diff
@@ -40,8 +40,24 @@ For developers:
...
+The Developer Portal supports built-in analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This feature gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable **Dashboard**. Each application has it's own dashboard that provides a high-level summary of **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 Latency by version
+* Error code distribution 
+
+All of these metrics can be configured across a time range up to **30 days**. 
+
+{:.note}
+> Free accounts will only have access to a time range of 24 hours.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/index.md:54]:
```suggestion
> Free accounts only have access to a time range of 24 hours.
```

File: app/konnect/dev-portal/index.md [hunk 3]
```diff
@@ -40,8 +40,24 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+The Developer Portal supports built-in analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This feature gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable **Dashboard**. Each application has it's own dashboard that provides a high-level summary of **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 Latency by version
```
Comment (lena-larionova) [inline app/konnect/dev-portal/index.md:48]:
```suggestion
* P99 latency by version
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: app/konnect/dev-portal/index.md [hunk 4]
```diff
@@ -40,8 +40,26 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard.
```
Comment (sichvoge) [inline app/konnect/dev-portal/index.md:45]: Fully customizable dashboards is not correct. Background and some headers change based on configurations made in the dev portal appearance but you can't change the tiles that host the visualizations. Those tiles stay with white background, black text, etc.

Example:
image

Page background is blue(ish).
Header is red(ish)
Tile background is still white.

File: app/konnect/dev-portal/index.md [hunk 5]
```diff
@@ -40,8 +40,26 @@ For developers:
...
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard. 
+
+Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 latency by version
+* Error code distribution 
+
+All of these metrics can be configured across a time range up to **30 days**.
```
Comment (sichvoge) [inline app/konnect/dev-portal/index.md:53]: Relative timeframe is up to "Last 30 days" but you can go back up to 90 days with custom.

File: app/konnect/dev-portal/customization/index.md [hunk 6]
```diff
@@ -20,6 +20,15 @@ From the **Appearance** menu you have the ability to modify the following option
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
+### Developer analytics dashboard
```
Comment (sichvoge) [inline app/konnect/dev-portal/customization/index.md:23]: Please remove this section. You can't customize in this version.

**Review by sichvoge — CHANGES_REQUESTED** [review]
Important comments before I can go ahead and approve this PR.

File: app/konnect/dev-portal/index.md [hunk 7]
```diff
@@ -40,8 +40,26 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a customizable dashboard.
```
Comment (sichvoge) [inline app/konnect/dev-portal/index.md:45]: I think we should avoid saying "customizable" in this sentence. Readers may think that that they can add/remove charts from this dashboard (which is referred to as "customizable dashboard").

**Review by sichvoge — CHANGES_REQUESTED** [review]
One last 💪

File: app/konnect/dev-portal/index.md (hunk 7 above)
Comment (Guaris) [inline app/konnect/dev-portal/index.md:45]:
```suggestion
You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs.
```

**Review by sichvoge — APPROVED** [review]
LGTM 👍

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**